json_schema_fuzz/schema_operations.py

```diff
@@ -0,0 +1,175 @@
+        })
+
+    # Strings
+
+    min_length = schema.get("minLength", None)
+    if min_length:
+        inverted_schemas.append({'maxLength': min_length - 1})
+
+    max_length = schema.get("maxLength", None)
+    if max_length:
+        inverted_schemas.append({'minLength': max_length + 1})
+
+    pattern = schema.get("pattern", None)
+    if pattern:
+        # Use a tempered greedy token that will match anything not matched
+        # by the original regex
+        # https://stackoverflow.com/questions/164414/how-to-inverse-match-with-regex
+        inverted_schemas.append({'pattern': f"^(?:(?!{pattern}).)*$"})
+
+    # Numbers
+
+    multiple_of = schema.get("multipleOf", None)
+    if multiple_of:
+        inverted_schemas.append({"notMultipleOf": multiple_of})
+
+    minimum = schema.get("minimum", None)
+    if minimum:
+        inverted_schemas.append({"exclusiveMaximum": minimum})
+    maximum = schema.get("maximum", None)
+    if maximum:
+        inverted_schemas.append({"exclusiveMinimum": maximum})
+    exclusive_minimum = schema.get("exclusiveMinimum", None)
+    if exclusive_minimum:
+        inverted_schemas.append({"maximum": exclusive_minimum})
+    exclusive_maximum = schema.get("exclusiveMaximum", None)
+    if exclusive_maximum:
+        inverted_schemas.append({"minimum": exclusive_maximum})
+
+    # Objects
+
+    properties = schema.get("properties", None)
+    if properties:
+        inverted_schemas.append({
+            "properties": {k: invert(v) for k, v in properties.items()},
+            "anyOf": [{"required": [k]} for k in properties.keys()],
+        })
+
+    required = schema.get("required", None)
+    if required:
+        inverted_schemas.append({
+            "anyOf": [{"disallow": [k] for k in properties.keys()}],
+        })
+
+    additional_properties = schema.get("additionalProperties", None)
+    if additional_properties:
+        inverted_schemas.append({
+            "anyAdditionalProperties": invert(additional_properties),
+        })
+
+    # Arrays
+
+    items = schema.get("items", None)
+    if items:
+        if isinstance(items, list):
+            item_conditions = []
+            for index, item_schema in enumerate(items):
+                item_conditions.append({
+                    "items": [{}] * index + [invert(item_schema)]
+                })
+            inverted_schemas.append(
+                {"anyOf": item_conditions + [{"maxItems": len(items)}]})
+        else:
+            inverted_schemas.append({"contains": invert(items)})
+
+    contains = schema.get("contains", None)
+    if contains:
+        inverted_schemas.append({"items": invert(contains)})
+
+    unique_items = schema.get("uniqueItems", None)
+    if unique_items:
+        inverted_schemas.append({"hasDuplicates": unique_items})
+
+    # Combine all schemas together and return
+
+    combined_schema = {}
+    for inverted_schema in inverted_schemas:
+        merge(combined_schema, inverted_schema, in_place=True)
```

Comment:
We don't need all of these to be true, we just need at least one to be true. I think we want something like
```python
combined_schema = {"anyOf": inverted_schemas}
```